src/scenes/home/home.js

```diff
@@ -213,10 +218,10 @@ class Home extends Component {
                 <Gala {...authProps} />
               )}
             />
-            <Route
+            {/* <Route
```

Comment:
This breaks the app from rendering locally. It needs a eslint disable next line rules to be uncommented.